### PR TITLE
Pull request for 2015-10-08

### PR DESCRIPTION
**Pull request by nkolban**
Work on #603 and #632

### CODE REVIEW

File: libs/network/esp8266/jswrap_esp8266.c [hunk 1]
```diff
@@ -354,10 +411,10 @@ void jswrap_ESP8266WiFi_onWiFiEvent(
   ) {
   // If the callback is null
   if (callback == NULL || jsvIsNull(callback)) {
-    if (jsWiFiEventCallback != NULL) {
-      jsvUnLock(jsWiFiEventCallback);
+    if (g_jsWiFiEventCallback != NULL) {
+      jsvUnLock(g_jsWiFiEventCallback);
```
Comment (gfwilliams) [inline libs/network/esp8266/jswrap_esp8266.c]: jsvUnLock is null-safe, so here (and in other places) you don't actually need the explicit pointer check - not that it does much harm though

**Comment by gfwilliams** [comment]
Thanks!

**Comment by MaBecker** [comment]
@nkolban can this be closed ?

**Comment by nkolban** [comment]
I don't think "pull requests" are closed ... they are just "merged" and thats the end of the path?

**Comment by MaBecker** [comment]
hmm - as I am not an expert on this I checked file jswrap_esp8266_network.c which has the changes you made.

gfwilliams has merged into espruino:master on 0 Oct 2015 - so this look complete to me.

What do you think ?

**Comment by nkolban** [comment]
I think it is complete ... its status says that it is "Merged" which in the set of "pull requests" is the equivalent in issues of it being closed?

**Pull request merged**